src/frustum_models/three_dimensional_lidar_frustum.cpp

```diff
@@ -100,8 +110,22 @@ bool ThreeDimensionalLidarFrustum::IsInside(const openvdb::Vec3d &pt)
   }
 
   // // Check if inside frustum valid vFOV
-  const double v_padded = fabs(transformed_pt[2]) + _vFOVPadding;
-  if (( v_padded * v_padded / radial_distance_squared) > _tan_vFOVhalf_squared)
+  // const double v_padded = fabs(transformed_pt[2]) + _vFOVPadding;
+  // if (( v_padded * v_padded / radial_distance_squared) > _tan_vSFOV_squared)
+  // {
+  //   return false;
+  // }
+  const double v_padded = transformed_pt[2] + _vFOVPadding;
```

Comment:
I don't think this necessarily works for the negative case, because the padding is always positive even when the transformed point coordinates are negative, that's why we have `fabs` above.